cmd/internal/apidump/apidump.go

```diff
@@ -184,9 +184,17 @@ func apidumpRunInternal(cmd *cobra.Command, _ []string) error {
 		}
 	}
 
+	agentRateLimit := 0.0
+	if commonApidumpFlags.RateLimit > 0.0 {
+		agentRateLimit = commonApidumpFlags.RateLimit
+	} else if envRateLimit := os.Getenv("POSTMAN_INSIGHTS_AGENT_RATE_LIMIT"); envRateLimit != "" {
+		if limit, err := strconv.ParseFloat(envRateLimit, 64); err == nil {
+			agentRateLimit = limit
+		}
+	}
 	// Rate limit must be greater than zero.
-	if commonApidumpFlags.RateLimit <= 0.0 {
-		commonApidumpFlags.RateLimit = 1000.0
+	if agentRateLimit <= 0.0 {
+		agentRateLimit = 1000.0
```

Comment:
Should this use `DefaultRateLimit` as well?  (We're probably going to want to bump it higher.)